src/main/java/nl/tudelft/jpacman/level/Level.java

```diff
@@ -353,18 +363,22 @@ public void run() {
 	 * 
 	 * @author Jeroen Roosen 
 	 */
-	public interface LevelObserver {
+	public interface WinObserver<T> {
 
 		/**
 		 * The level has been won. Typically the level should be stopped when
 		 * this event is received.
 		 */
-		void levelWon();
+		void levelWon(T t);
 
 		/**
 		 * The level has been lost. Typically the level should be stopped when
 		 * this event is received.
 		 */
-		void levelLost();
+		void levelLost(T t);
+	}
+
+	public interface LevelObserver extends WinObserver<Level> {
```

Comment:
What is exactly the purpose of this empty interface? Why can't these two be merged?